### PR TITLE
[windows] [process events] receive environment block from driver in process events

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?

Adds the ability to receive the environment block in the security probe, which includes
the environment block in process events.

By doing so, USM will now add the service tag if it's set in the environment.

### Motivation

Allow USM to work with tags from environment

### Describe how to test/QA your changes

Automated tests included.

### CODE REVIEW

**Review by hmahmood — APPROVED** [review]
The extra line added to monitor.go approved

**Review by paulcacheux — APPROVED** [review]

File: test/new-e2e/tests/sysprobe-functional/apmtags_test.go [hunk 1]
```diff
@@ -315,7 +316,19 @@ func (v *apmvmSuite) TestUSMAutoTaggingSuite() {
...
-			localcmd := fmt.Sprintf(pscommand, testScript, targetport, targetpath, strings.Join(test.expectedClientTags, ","), strings.Join(test.expectedServerTags, ","), testExe)
+			var envstring string
+			for k, v := range test.clientEnvVars {
+				envstring += fmt.Sprintf("$Env:%s=\"%s\" ; ", k, v)
+			}
+			localcmd := fmt.Sprintf(pscommand, envstring, testScript, targetport, targetpath, strings.Join(test.expectedClientTags, ","), strings.Join(test.expectedServerTags, ","), testExe)
+
+			if len(test.clientEnvVars) > 0 {
+				var envarg string
+				for k, v := range test.clientEnvVars {
```
Comment (alexn-dd) [inline test/new-e2e/tests/sysprobe-functional/apmtags_test.go:327]: What is the purpose of this loop?
The formatted string is local to this scope only.

**Review by alexn-dd — APPROVED** [review]

File: test/new-e2e/tests/sysprobe-functional/apmtags_test.go (hunk 1 above)
Comment (derekwbrown) [reply test/new-e2e/tests/sysprobe-functional/apmtags_test.go:330]: > What is the purpose of this loop? The formatted string is local to this scope only.

it iterates the loop of predefined tests.
Each time the loop iterates, there's a (slightly) different command line that's generated.

**Pull request merged**